neurovault/apps/statmaps/views.py

```diff
@@ -1002,7 +1002,7 @@ def find_similar_json(request, pk, collection_cid=None):
 
     image1 = get_image(pk, None, request)
     pk = int(pk)
-    max_results = 100
+    max_results = Image.objects.all().count()  # image number will always be higher than comparison number
```

Comment:
Just the json, not the html. We said the comparison should be over the whole set and not just the first 100, but we can limit to the first 100, no probs.